src/cpipes.c

```diff
@@ -24,6 +25,7 @@
 
 void interrupt_signal(int param);
 void parse_options(int argc, char **argv);
+static void find_num_colours(int argc, char **argv);
```

Comment:
Before this commit `grep -o color src/*.{c,h} | wc -l` shows 2 occurrences for `color` and 18 for `colour`; as of 2dc1121, 136 for `color` and 88 for `colour`.

I think it's better you use only one spelling in your code regardless how ncurses spells it.